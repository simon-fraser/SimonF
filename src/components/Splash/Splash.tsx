import bg from '../../assets/DSC_0591_4k.jpg'
console.log(bg)

function Splash() {
  return (<figure aria-hidden="true" aria-description='Background image for web page' className="absolute bg-cover bg-top contrast-[1.1] inset-px z-0" style={{ backgroundImage: `url(${bg})` }} />)
}

export default Splash
