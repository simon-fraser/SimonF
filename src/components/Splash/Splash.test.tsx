import { render } from '@testing-library/react';
import { test, describe, expect } from 'vitest';
import React from 'react';
import Splash from './Splash';

describe('components/Splash', () => {
  test('will render correctly',() => {
    const {asFragment} = render(<Splash />)

    expect(asFragment()).toMatchSnapshot();
  })
})