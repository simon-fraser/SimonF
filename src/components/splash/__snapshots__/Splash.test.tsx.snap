// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components/splash > will render correctly 1`] = `
<DocumentFragment>
  <figure
    aria-description="Background image for web page"
    aria-hidden="true"
    assets=""
    class="absolute bg-cover bg-top contrast-[1.1] inset-px z-0"
    dsc_0591_4k.webp=""
    src=""
    style="background-image: url("
  />
</DocumentFragment>
`;
